hydra_agent/partial_collection_crawler.py

```diff
@@ -0,0 +1,95 @@
+from requests import get
+
+
+class PartialCollectionCrawler:
+    """
+    Crawler for Crawling Collections. 
+    It can move forwards, backwards or can jump to specific page
+    """
+
+    def __init__(self, response):
+        self.response = response
+        self.base_url = 'http://localhost:8080'
+
+    def initialize_forward(self):
+        """
+        Initializes the crawler to move forwards.
+        :returns: Iterator
+        """
+        view = self.response['view']
+        present_page = view['@id']
+        while True:
+            self.response = get(self.base_url + present_page).json()
+            yield self.response['members']
+            view = self.response['view']
+            print('view')
+            if 'next' in view:
+                present_page = view['next']
+            else:
+                raise StopIteration("No more collection pages")
+
+    def initialize_backward(self):
+        """
+        Initializes the crawler to move backwards.
+        :returns: Iterator
+        """
+        view = self.response['view']
+        present_page = view['@id']
+        while True:
+            self.response = get(self.base_url+present_page).json()
+            yield self.response['members']
+            view = self.response['view']
+            if 'previous' in view:
+                present_page = view['previous']
+            else:
+                raise StopIteration("No more collection pages")
+
+    def jumpToPage(self, page):
```

Comment:
please follow the same naming conventions for all the functions. Should be `jump_to_page`